controlalgorithm/heat_mgmt_algorithm.py

```diff
@@ -154,9 +154,9 @@ def get_solar_angle_weight():
 Output:
 tilt_angle_final (float): final tilt angle for maximum energy efficiency
 """
-def heat_mgmt_algorithm(ac_t):
```

Comment:
Not sure about this one, we might lose some testability here because now the temperature comes from a function we cant change. Up to you though. It's reasonable to do either

Reply:
I'm not sure that's much of an issue since you can just mock get_int_temp() or provide two implementations

Reply:
Okay, fair enough.

Reply:
Unit Tests for this are to be fixed in another PR